package.json

```diff
@@ -18,19 +18,24 @@
   },
   "repository": {
     "type": "git",
-    "url": "git://github.com/neo4j/neo4j-javascript-driver.git"
+    "url": "git://github.com/neo4j-drivers/experimental-query-api-wrapper.git"
   },
   "keywords": [
     "http",
     "neo4j",
     "driver"
   ],
   "author": "Antonio Barcélos",
```

Comment:
Maybe the author field can be the same in we have in the driver, I don't remember which value is.